### DISCUSSION TITLE
ClusterConfigAuditReport not generated

### DISCUSSION THREAD

**Discussion by r0binak** [aquasecurity/trivy-operator#746] [Help & Support]
I installed trivy-operator with default settings in the cluster and I expect that a ClusterConfigAuditReport report will be generated with compliance checks for CIS Kubernetes, but this does not happen. Why is that?

**Comment by chen-keinan** [comment]
@r0binak currently the `trivy-operator` do not support CIS Kubernetes report , in `trivy-operator` v0.7.0 we have added [Infra Assessment checks](https://aquasecurity.github.io/trivy-operator/v0.7.1/crds/infraassessment-report/) , which include most of the `CIS Kubernetes` checks (everything which do not require access to FS) the rest is planned to be added later
